fontir/src/ir.rs

```diff
@@ -1059,6 +1064,11 @@ impl AnchorKind {
         }
         Ok(AnchorKind::Base(name.into()))
     }
+
+    /// Returns `true` If this is a base or ligature base anchor
```

Comment:
describing what an attaching anchor is might be more helpful, e.g. what attaches to what